manifests/init.pp

```diff
@@ -24,6 +24,14 @@
     'RedHat': {
       $init_template             = 'tacacsplus/tac_plus-redhat-init.erb'
       $tac_plus_template_default = 'tacacsplus/tac_plus.conf.erb'
+      $tac_plus_service          = 'tac_plus'
+      $tac_plus_config           = '/etc/tac_plus.conf'
+    }
+    'Debian': {
```

Comment:
Debian will not only match on Ubuntu systems. To be more precise it is needed to change the case condition to from ```$::osfamily``` to ```$::operatingsystem```.